### ISSUE TITLE
Unable to add "Missing" resources from a table when unowned

### ISSUE THREAD

**Issue by Elliander** [StrangeLoopGames/EcoIssues#7209]
Found an issue where I could not add the missing resources to any table after the job has started working if I am not the owner of it. In this case, I attempted to make a mint using an Unowned Anvil. I had the bricks and lumber handy, but because I couldn't hold both at once I had to start with one. Afterwards, however, I could not proceed to add it. No error message, it just doesn't accept it. I am unsure if the behavior is the same if another player owns it, as this was only observed for unowned - created by another player outside of their property.

**Comment by Elliander** [comment]
I did some additional testing last night, and found that when I returned to my own property I also couldn't add missing resources by dragging over it, but could by right clicking over the table while holding the resource. It's possible, then, that the issue is unrelated to ownership and maybe for whatever reason it stopped working properly while the game was running. Never had it happen before trying to run an unowned table though.
